index.js

```diff
@@ -0,0 +1,74 @@
+//CALCULATIONS
+
+//UNIT1
+
+const hexToRGB = (hexColor) => {
+    const R = parseInt(hexColor.slice(1, 3), 16);
+    const G = parseInt(hexColor.slice(3, 5), 16);
+    const B = parseInt(hexColor.slice(5), 16);
+    return [R, G, B];
+  }
+
+//UNIT2
+  
+const getLuminance = (RGBarray) => {
+    //convert 8bit colors to
+    //RsRGB, GsRGB BsRGB
+    const XsRGBarray = RGBarray.map(elem => elem / 255);
+
+    //use function to get r, g, b for final formula
+    //UNIT2.1
+    const getRGBCoefficient = (XsRGB) => {
+    if (XsRGB <= 0.03928) {
+        return XsRGB / 12.92;
+    }
+    else {
+        return Math.pow((XsRGB + 0.055) / 1.055, 2.4);
+    }
+    }
```

Comment:
This curly brace should be two spaces to the left for proper indentation.
We will be adding a formatter, which will catch these formatting issues.
But for now, please make that change.